modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -73,13 +73,14 @@ $ oc describe packagemanifests <operator_name> -n openshift-marketplace
 
 . An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
 +
-The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces`, then the `openshift-operators` namespace already has an appropriate Operator group in place.
+The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the `AllNamespaces` or `SingleNamespace` mode. If the Operator you intend to install uses the `AllNamespaces` mode, then the `openshift-operators` namespace already has the appropriate `global-operators` Operator group in place.
```

Comment:
Don't really need the "then" here. ISG discourages it.

Reply:
Good call. It also reads better without it.